ToDoList_Project/ToDoList_Project/Controllers/TasksController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Org.BouncyCastle.Bcpg;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using ToDoList_Project.DTOs;
+using ToDoList_Project.Repository;
+
+namespace ToDoList_Project.Controllers
+{
+    [Route("api/[controller]")]
+    [Authorize]
+    [ApiController]
+
+    public class TasksController : ControllerBase
+    {
+        private readonly ITasksRepository tasksRepository;
+        public TasksController(ITasksRepository tasksRepository)
+        {
+            this.tasksRepository = tasksRepository;
+        }
+        [HttpGet("AllTasks")]
```

Comment:
```suggestion
        [HttpGet("")]
```
thêm filter dùng query param nữa nhé

Reply:
đọc lại tài liệu về Restful API để thiết kế path cho chuẩn nhé